core/src/main/java/io/th0rgal/oraxen/config/ConfigsManager.java

```diff
@@ -108,6 +110,15 @@ public void validatesConfig() {
         String languageFile = "languages/" + Settings.PLUGIN_LANGUAGE + ".yml";
         language = validate(resourcesManager, languageFile, defaultLanguage);
 
+        // check bbModelFolder
+        bbModelFolder = new File(plugin.getDataFolder(), "bbmodels");
```

Comment:
Think i would rather have this be under pack folder, same as "external_packs", make a "bbmodels" folder in there